src/app/sign_up/page.tsx

```
@@ -0,0 +1,3 @@
+import { SignUp } from "../../feature/auth/pages/sign_up";
+
+export default SignUp as React.ComponentType<{}>;
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)